pkg/csi/service/common/util.go

```diff
@@ -354,6 +355,7 @@ func ConvertVolumeHealthStatus(volHealthStatus string) (string, error) {
 		// This implies the volume does not exist any more.
 		// Set health annotation to "Inaccessible" so that
 		// the caller can make appropriate reactions based on this status
+		log.Debug("SPBM does not set health status, volHealthStatus is empty.")
```

Comment:
How do I know for which volume the health status was not set?

Reply:
Print volume id in log.